moodler/moodle.py

```diff
@@ -67,23 +82,25 @@ def submissions_statistics(course_id, is_verbose=False, download_folder=None):
     total_submissions = 0
     total_ungraded = 0
     total_resubmissions = 0
+    total_unreleased = 0
     assignments_statistics = {}
 
     assignments = get_assignments(course_id)
     users_map = get_students(course_id)
 
     for assignment in assignments:
-        current_assignment_submissions_amount = len(assignment.submissions)
-        current_assignment_ungraded = assignment.ungraded()
-        current_assignment_ungraded_amount = len(current_assignment_ungraded)
-
-        current_assignment_resubmissions_amount = 0
+        submissions_amount = len(assignment.submissions)
```

Comment:
do we need to count ALL submissions or submitted only?

Reply:
We need all submissions. This way we can easily see if all the students submitted the exercise at least once, or if there is an exercise that the students have difficulty with.